<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowsFour</class>
 <widget class="QMainWindow" name="WindowsFour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_donneur">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>modifier_donneur</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>461</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>modifier</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ancienId</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nouveauId</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>nouveauNumTele</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nouvelleQteDon</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nouvelleAdresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nouveauMail</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_ancien">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_nouveau">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num_tele_nouveau">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_qte_don_nouvelle">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adresse_nouvelle">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mail_nouveau">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2">
     <attribute name="title">
      <string>Ajouter_un_donneur</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>441</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>num_tele</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adresse">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mail">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num_tele">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_qte_don">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>qte_don</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3">
     <attribute name="title">
      <string>Afficher_des_donneur</string>
     </attribute>
     <widget class="QTableView" name="tab_don">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_1">
     <attribute name="title">
      <string>Supprimer_un_donneur</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>33</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifient</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_sup">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>trier_les_donneur</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView2">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>61</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>rechercher_donneur</string>
     </attribute>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>exporte_client</string>
     </attribute>
     <widget class="QPushButton" name="pdf_client">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>client</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>exporter unpdf</string>
     </attribute>
     <widget class="QPushButton" name="pb_pdf">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>donneur</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>ajouter_client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>401</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>ajout_client</string>
      </property>
      <widget class="QLabel" name="id_client">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id_client</string>
       </property>
      </widget>
      <widget class="QLabel" name="nom_client">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nom_client</string>
       </property>
      </widget>
      <widget class="QLabel" name="prenom_client">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="mail_client">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="num_tele_client">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>num_tele_client</string>
       </property>
      </widget>
      <widget class="QLabel" name="qte_don_client">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>qte_achat</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_client">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom_client">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom_client">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mail_client">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num_tele_client">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_qte_achat_client">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_client">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adresse_client">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>supprimer_client</string>
     </attribute>
     <widget class="QLabel" name="id_sup_client_label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_sup_client">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer_client">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>afficher_client</string>
     </attribute>
     <widget class="QTableView" name="tab_don_client">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>modifier_client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>461</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>modifier</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ancienId</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nouveauId</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>nouveauNumTele</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nouvelleQteDon</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nouvelleAdresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nouveauMail</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_ancien_client">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_nouveau_client">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num_tele_nouveau_client">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_qte_achat_nouvelle_client">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adresse_nouvelle_client">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mail_nouveau_client">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_client">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>trier_client</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView2_client">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>rechercher_client</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonLogout">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
